modules/blogging/src/Volo.Blogging.MongoDB/Volo/Blogging/Posts/MongoPostRepository.cs

```diff
@@ -58,5 +58,13 @@ public async Task<List<Post>> GetOrderedList(Guid blogId, bool @descending = fal
 
             return await query.OrderByDescending(x => x.CreationTime).ToListAsync(GetCancellationToken(cancellationToken));
         }
+
+        public async Task<List<Post>> GetListByUserIdAsync(Guid userId, CancellationToken cancellationToken = default)
+        {
+            var query = (await GetMongoQueryableAsync(cancellationToken)).Where(x => x.CreatorId == userId)
+                .OrderBy(x => x.CreationTime);
```

Comment:
On the EF Core side, you are getting posts by newest, but in MongoDB implementation it's the opposite. So, please make the following change:

```suggestion
                .OrderByDescending(x => x.CreationTime);
```

Reply:
Same with https://github.com/abpframework/abp/pull/16313#discussion_r1176109802